playwright/api/call-blockfrost/blockfrost.api.ts

```diff
@@ -9,7 +9,7 @@ export function blockfrostApi(request: APIRequestContext) {
   const getLatestBlockData = async () => {
     return BaseApi.getData(
       request,
-      Endpoint.BlockFrost.LastBlockData.Base,
+      Endpoint.BlockFrost.BLOCKS.Base + "/latest",
```

Comment:
/latest should be in endpoints.ts - why? becasue you want central place to handle endpoints as per comment in endpints.ts